site/app/templates/grading/electronic/Details.twig

```diff
@@ -337,9 +343,9 @@
             {% if graded_gradeable.getTaGradedGradeable().hasVersionConflict() %}
                 {% set contents = "Version Conflict" %}
                 {% set btn_class = "btn-primary" %}
-            {% elseif not graded_gradeable.isTaGradingComplete() %}
+            {% elseif not graded_gradeable.isTaGradingComplete() and core.getUser().getGroup() < 4 or core.getUser().getGroup() == 4 and not graded_gradeable.isPeerGradingComplete() %}
```

Comment:
Add some parentheses to your logic.

Reply:
Also, make sure that this looks right for students and instructors (I think it might make peer gradeables look ungraded for instructors)